src/utils.js

```diff
@@ -175,6 +177,73 @@ const validateEmailTemplateFields = (formData) => {
   return errors;
 };
 
+const validateEmailAddresses = (emails) => {
+  const result = {
+    validEmails: [],
+    validEmailIndices: [],
+    invalidEmails: [],
+    invalidEmailIndices: [],
+  };
+  if (!emails) {
```

Comment:
is this if statement necessary? in the case of an empty list wouldn't the forEach not do anything and return result anyways?

Reply:
@chavesj You are correct that for an empty list, the forEach would just return result anyways. Though, what if `emails` is null? `emails.forEach` when `emails` is null would return in a `TypeError: can't access property "forEach" of null`.